packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -231,6 +232,16 @@ abstract class MultiDragGestureRecognizer extends GestureRecognizer {
   /// [Drag] object returned by this callback.
   GestureMultiDragStartCallback? onStart;
 
+  /// Called when interaction starts. This limits the dragging behavior
+  /// for custom clicks (such as scroll click).
+  /// When value is `(int buttons) => false`, allow no interactions.
+  /// When value is `(int buttons) => true`, allow all interactions.
+  /// When value is `(int buttons) => buttons & kPrimaryButton != 0`, allow
+  /// if selection contains a primary button.
+  ///
+  /// Defaults to `(int buttons) => buttons & kPrimaryButton != 0`.
+  final bool Function(int buttons)? inputPointerBehavior;
```

Comment:
Can you add a typedef for this so we can have a name for the predicate?

Reply:
Yes, or pass the pointer and call `pointer.buttons` so the user has more data.

Reply:
[Style Guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables): consider adding a typedef. This might improve the readability.

Reply:
`isPointerAllowed` sounds good to me. https://stackoverflow.com/questions/8548232/whats-the-best-way-to-name-a-predicate-function